modules/web/js/ballerina/components/package-definition.jsx

```diff
@@ -142,10 +142,18 @@ class PackageDefinition extends React.Component {
      * @param {string} value - name of the package newly imported
      */
     handleAddImport(value) {
-        const newImportDeclaration = BallerinaASTFactory.createImportDeclaration();
-        newImportDeclaration.setPackageName(value);
-        newImportDeclaration.setParent(this.props.model.parent, { doSilently: true });
-        this.props.model.parent.addImport(newImportDeclaration);
+        if (!value) {
+            return;
+        }
+        value = 'import ' + value + ';\n';
+        parseContent(value)
+            .then((jsonTree) => {
+                if (!jsonTree.root[1]) {
```

Comment:
Wont it make sense if we have if (jsonTree.root[1]) { 

We can avoid the return statement as well.